### PR TITLE
feat(pds-ember): add Pds::Banner component

### PR DESCRIPTION
**Pull request by CITguy**
JIRA: PDS-189

### Screenshots
_info_
banner - info

_info-no-border_
banner - info (borderless)

_success_
banner - success

_warning_
banner - warning

_error_
banner - error

### CODE REVIEW

File: packages/pds-ember/addon/components/pds/banner/index.js [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+  }
+
+  get iconType() {
+    let variant = this.variant
+    let result = ICON_TYPES[variant]
+
+    console.log('variant', variant)
+    console.log('ICON_TYPES', ICON_TYPES)
+    console.log('result', result)
+
```
Comment (CITguy) [inline packages/pds-ember/addon/components/pds/banner/index.js:34]: remove console statements

File: packages/pds-ember/addon/components/pds/banner/utils.js [hunk 2]
```diff
@@ -0,0 +1,9 @@
+export const DEFAULT_ICON = 'info'
```
Comment (CITguy) [inline packages/pds-ember/addon/components/pds/banner/utils.js:1]: s/ICON/VARIANT/

**Review by CITguy — COMMENTED** [review]
see comments for changes

**Review by venusang — APPROVED** [review]
As always, really appreciate your documentation.  Vercel preview is 👍 .  I have no real feedback outside of looks good.

**Pull request merged**